pypeit/core/extract.py

```diff
@@ -1202,7 +1202,7 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, maxdev
     spat_vec = np.arange(nspat)
     slit_spec_pos = nspec/2.0
 
-    slit_spat_pos = (np.interp(slit_spec_pos, spec_vec, slit_left), np.interp(slit_spec_pos, spec_vec, slit_righ))
+    #slit_spat_pos = (np.interp(slit_spec_pos, spec_vec, slit_left), np.interp(slit_spec_pos, spec_vec, slit_righ))
```

Comment:
Should this really be commented out?

Reply:
I've put it back in for now, but this variable is not used in the code (nor is `slit_spec_pos`, on line 1203, for that matter). I presumed that this was some deprecated functionality.